packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -239,11 +239,11 @@ export default defineComponent({
     const { getOptionByValue, getValue, getText, getTrackBy } = useSelectableList(props)
 
     const {
-      validate,
       isFocused,
-      computedErrorMessages,
+      validate,
       computedError,
-    } = useFormComponent(props, () => reset(), () => focus(), () => blur())
+      computedErrorMessages,
+    } = useValidation(props, context.emit, () => reset(), () => focus(), () => blur())
```

Comment:
Here we can make:
```ts
setup (props, { emit, expose }) {
  // ...
}
```

```suggestion
    } = useValidation(props, emit, () => reset(), () => focus(), () => blur())
```

(You can ignore that comment)

Reply:
Fixed it